.circleci/generate_artifacts.sh

```diff
@@ -52,33 +42,53 @@ $BUILTPATH/ddev_gen_autocomplete
-tar -czf $ARTIFACTS/ddev_macos.$VERSION.tar.gz ddev *completion*.sh mkcert
-
-# Generate linux tarball/zipball
-cd $BASE_DIR/.gotmp/bin
-curl -sSL -o mkcert https://github.com/FiloSottile/mkcert/releases/download/${MKCERT_VERSION}/mkcert-${MKCERT_VERSION}-linux-amd64 && chmod +x mkcert
-tar -czf $ARTIFACTS/ddev_linux.$VERSION.tar.gz ddev *completion*.sh mkcert
-
-# generate windows tarball/zipball
-cd $BASE_DIR/.gotmp/bin/windows_amd64
-curl -sSL -o mkcert.exe https://github.com/FiloSottile/mkcert/releases/download/${MKCERT_VERSION}/mkcert-${MKCERT_VERSION}-windows-amd64.exe
-tar -czf $ARTIFACTS/ddev_windows.$VERSION.tar.gz ddev.exe *completion*.sh mkcert.exe
-zip $ARTIFACTS/ddev_windows.$VERSION.zip ddev.exe *completion*.sh
+# Create tarball of completion scripts
+pushd .gotmp/bin >/dev/null && tar -czf $ARTIFACTS/ddev_shell_completion_scripts.$VERSION.tar.gz *completion*.sh && popd >/dev/null
+cp $BASE_DIR/.gotmp/bin/windows_amd64/ddev_windows_installer*.exe $ARTIFACTS
+
+# Generate macOS-amd64 tarball/zipball
+pushd $BASE_DIR/.gotmp/bin/darwin_amd64
+curl -sSL -o mkcert https://github.com/drud/mkcert/releases/download/${MKCERT_VERSION}/mkcert-${MKCERT_VERSION}-darwin-amd64 && chmod +x mkcert
+tar -czf $ARTIFACTS/ddev_macos-amd64.$VERSION.tar.gz ddev *completion*.sh mkcert
+popd
+
+# Generate linux-amd64 tarball/zipball
+pushd $BASE_DIR/.gotmp/bin/linux_amd64
+curl -sSL -o mkcert https://github.com/drud/mkcert/releases/download/${MKCERT_VERSION}/mkcert-${MKCERT_VERSION}-linux-amd64 && chmod +x mkcert
+tar -czf $ARTIFACTS/ddev_linux-amd64.$VERSION.tar.gz ddev *completion*.sh mkcert
+popd
+
+# Generate linux-arm64 tarball/zipball
+pushd $BASE_DIR/.gotmp/bin/linux_arm64
+curl -sSL -o mkcert https://github.com/drud/mkcert/releases/download/${MKCERT_VERSION}/mkcert-${MKCERT_VERSION}-linux-arm64 && chmod +x mkcert
+tar -czf $ARTIFACTS/ddev_linux-arm64.$VERSION.tar.gz ddev *completion*.sh
+popd
+
+# Generate linux-arm tarball/zipball
+pushd $BASE_DIR/.gotmp/bin/linux_arm
+curl -sSL -o mkcert https://github.com/drud/mkcert/releases/download/${MKCERT_VERSION}/mkcert-${MKCERT_VERSION}-linux-arm && chmod +x mkcert
+tar -czf $ARTIFACTS/ddev_linux-arm.$VERSION.tar.gz ddev *completion*.sh
+popd
+
+# generate windows-amd64 tarball/zipball
+pushd $BASE_DIR/.gotmp/bin/windows_amd64
+curl -sSL -o mkcert.exe https://github.com/drud/mkcert/releases/download/${MKCERT_VERSION}/mkcert-${MKCERT_VERSION}-windows-amd64.exe
+tar -czf $ARTIFACTS/ddev_windows-amd64.$VERSION.tar.gz ddev.exe *completion*.sh mkcert.exe
 if [ -d chocolatey ]; then
-    tar -czf $ARTIFACTS/ddev_chocolatey.$VERSION.tar.gz chocolatey
+    tar -czf $ARTIFACTS/ddev_chocolatey_amd64-.$VERSION.tar.gz chocolatey
 fi
+popd
 
-cp ddev_windows_installer*.exe $ARTIFACTS
+# generate windows-arm tarball/zipball
+pushd $BASE_DIR/.gotmp/bin/windows_arm
+curl -sSL -o mkcert.exe https://github.com/drud/mkcert/releases/download/${MKCERT_VERSION}/mkcert-${MKCERT_VERSION}-windows-arm.exe
```

Comment:
`windows-arm` is not part of the 1.4.4 release: https://github.com/drud/mkcert/releases. I tried with `windows-arm` but it doesn't seem to work on arm64. i386 (32-bit) does work however (32-bit emulation on arm64). So please:

- Update mkcert version to v1.4.6 (just released)
- use `-windows-386.exe` on windows-arm :)

Reply:
So you want windows/386 (See `go tool dist list`) ? I wonder if that will work without building 32-bit versions of the images? It may not be worth fighting this, of course. I was hoping that the windows-arm would work on your machine. 

I agree about pull windows/arm out of there. But do you think it's useful to have windows/i386?

Reply:
That would be windows/386 indeed. Thing is, Windows arm64 doesn't even have the ability to run Docker Desktop, so it's actually a waste of effort for now. Users of Windows arm64 will need to be doing everything from within WSL2, so they'll be using DDEV Linux arm64 binaries in that case. Let's just remove windows_arm and re-consider in a few months 👍

Reply:
Oh, I guess I didn't understand how you're running docker on your surface device. You must be running Linux docker inside WSL2?

I've removed windows/arm

Reply:
Yes, correct :) no other way to run Docker currently on arm64 Windows devices 😊

Reply:
I sure hear another blog post coming! :)

Reply:
Haha, so much to blog about after this whole experience! Def will write something soon 🚀

Reply:
We may want to collaborate on a single blog that explains windows/arm64, linux/arm64, raspberry pi/arm64, raspberry pi/arm. /cc @ekl1773

Reply:
Yes sounds good!